lib/pejs.ts

```diff
@@ -97,44 +97,45 @@ export class PEJS {
   }
 
   /**
-   * This method helps create a submittablePresentation. A submittablePresentation after signing becomes verifiablePresentation. And can be sent to
+   * This method helps create a submittablePresentation. A submittablePresentation after signing becomes types. And can be sent to
```

Comment:
unexpected edit, I think.
we can revert this.

Reply:
fixed and updated wording a bit